docs_espressif/en/additionalfeatures/hints-viewer.rst

```diff
@@ -0,0 +1,25 @@
+Hints Viewer
```

Comment:
It would be great if you could add a Hints Viewer link in the Build project as a reference. Not only this, but all relevant links can be added there, and this is applicable to all features.